docs/src/examples/components/Menu/commonSteps.ts

```diff
@@ -0,0 +1,30 @@
+import { Menu } from '@stardust-ui/react'
+
+interface StepsOptions {
+  vertical?: boolean
+}
+
+const menuSelector = `.${Menu.className}`
+const selectors = {
+  menu: menuSelector,
+  item: (itemIndex: number) => `${menuSelector} li:nth-child(${itemIndex}) a`,
```

Comment:
```suggestion
  menu: `.${Menu.className}`,
  item: (itemIndex: number) => `.${Menu.className} li:nth-child(${itemIndex}) a`,
```

We can omit `menuSelector` as it doesn't make selectors more readable

Reply:
I'd argue it doesn't make it less readable either, so why not reuse. It's really NIT so I'd rather not change it